settings.env.example

```diff
@@ -0,0 +1,17 @@
+ZOPE_URL=http://www.eionet.europa.eu
+
+GEMET_URL=/gemet
+
+# FORCE_SCRIPT_NAME=/gemet
+
+# Fill in the gemet.rdf.gz file path in order to serve it at http://example.com/gemet/gemet.rdf.gz
+# GEMET_RDFGZ_PATH=/path/to/gemet.rdf.gz
+
+
+# Fill in the following settings to enable LDAP authentication
+AUTH_LDAP_SERVER_URI=ldap://ldap3.eionet.europa.eu
+AUTH_LDAP_USER_DN_TEMPLATE=uid=%(user)s,ou=Users,o=EIONET,l=Europe
+AUTH_LDAP_START_TLS=True
+AUTH_LDAP_USER_FIRST_NAME=givenName
+AUTH_LDAP_USER_LAST_NAME=sn
+AUTH_LDAP_USER_EMAIL=mail
```

Comment:
We can consider these four variables (`AUTH_LDAP_START_TLS`, `AUTH_LDAP_USER_FIRST_NAME`, `AUTH_LDAP_USER_LAST_NAME`, `AUTH_LDAP_USER_EMAIL`) constants (they don't need to be modified for each installation). The other two (`AUTH_LDAP_SERVER_URI ` and `AUTH_LDAP_USER_DN_TEMPLATE`) you should be fetched from the environment.